db/schema.rb

```diff
@@ -86,6 +86,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "admin", default: false
+    t.integer "role"
```

Comment:
This integer role does not belong here -- that is an artifact left over from when we were going to use pundit.  I think you can just delete that line from schema.rb, commit and push the change, and that will correct it.

Reply:
Change made.  New updated schema file uploaded to branch.